### ISSUE TITLE
Can't inline script such as `const x = '</script>'`

### ISSUE THREAD

**Issue by seiyab** [BjornTheProgrammer/bun-plugin-html#4]
Hi. Trying to this interesting and useful plugin, I found inlining scripts that contains `</script>` enexpectedly closes script tag.

Adding a context, I'm trying to bundle React application into a single HTML.
React (probably react-dom) contains `div.innerHTML = "<script></script>";`, at least as of 18.2.0.

**Comment by seiyab** [comment]
html-inline-script-webpack-plugin probably handles it like this:
```ts
(asset.source() as string).replace(/(<)(\/script>)/g, '\\x3C$2')
```
https://github.com/icelam/html-inline-script-webpack-plugin/blob/fd19ece98f3f1b13a02a61dd481f50b31cb8945d/src/HtmlInlineScriptPlugin.ts#L98

**Comment by BjornTheProgrammer** [comment]
Would you be willing to make a minimum reproducible example of this bug?

**Comment by seiyab** [comment]
OK, I will. Which do you prefer?

- Really minimal one that makes sence little.
    - e.g. `window.alert("</script>")`
- More practical one
    - e.g. bundling React

**Comment by BjornTheProgrammer** [comment]
The more practical one with a minimal React example. Also, note that the code has been significantly reworked in v1.3.1, so try upgrading first.

**Comment by seiyab** [comment]
Here is.
https://github.com/seiyab/repro/tree/main/bun-plugin-html-inline

**Comment by BjornTheProgrammer** [comment]
@seiyab this should now be fixed as of v1.3.2. Let me know if you have any other issues.

**Comment by seiyab** [comment]
I confirmed v1.3.2 works fine 👍 🎉 🥇
